pkgs/development/python-modules/napari-console/default.nix

```diff
@@ -12,16 +12,16 @@
 
 buildPythonPackage rec {
   pname = "napari-console";
-  version = "0.0.7";
-  format = "pyproject";
+  version = "0.0.9";
+  pyproject = true;
 
   disabled = pythonOlder "3.8";
 
   src = fetchFromGitHub {
     owner = "napari";
-    repo = pname;
+    repo = "napari-console";
```

Comment:
I don't mind, but it's a redundant diff (did `nix-update` do that?)

Reply:
https://github.com/NixOS/nixpkgs/issues/277994

Reply:
Yea, reason enough